.github/ISSUE_TEMPLATE/release-template.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Release template
+about: Internally used for new releases
+title: Release x.y.z
+labels: 'Type: Maintenance'
+assignees: jblz, GaryJones, mikeyarce, hbbtstar
+
+---
+
+:warning: DO NOT MERGE (YET) :warning:
+
+[Remaining work for this Milestone]``(https://github.com/Parsely/wp-parsely/milestone/14)
```

Comment:
Did those backticks sneak in or do they serve a purpose?

```suggestion
[Remaining work for this Milestone](https://github.com/Parsely/wp-parsely/milestone/MILESTONE_ID_HERE)
```

Reply:
I blame my cat.